### ISSUE TITLE
support? log exp function

### ISSUE THREAD

**Issue by birdol** [danhunsaker/bcmath#4]
when I use php function ,like ln() log()、exp(), in $formula with bc::parse(), it output a string for me.
also in pow(x,y)

**Comment by danhunsaker** [comment]
Yes, `BC::parse()` doesn't support function calls. You have to do those other ways, at least for the moment.

At some point, I may create/release a complete mathematics parsing/evaluation library. It will be a separate package from this one, though, even if it does end up using BC::math behind the scenes.
